py/src/ai/h2o/sparkling/SharedBackendConf.py

```diff
@@ -410,10 +411,12 @@ def setIgnoreSparkPublicDNSDisabled(self):
         return self
 
     def setClientWebEnabled(self):
+        warnings.warn("Method 'setClientWebEnabled' is deprecated and will be removed in the next major release 3.30.")
         self._jconf.setClientWebEnabled()
```

Comment:
Do we have to call the  scala property here?

Reply:
👍 nope